.github/CONTRIBUTING.md

```diff
@@ -75,6 +75,15 @@ As a general rule, if a repository A is a submodule dependency of repository B t
 
 - [concordium-reference-wallet-ios](https://github.com/Concordium/concordium-reference-wallet-ios)
   This is a reference implementation of a mobile wallet for the iOS platform.
+  
+- [concordium-desktop-wallet](https://github.com/Concordium/concordium-desktop-wallet)
+  This is an implementation of a desktop wallet for Windows, macOS and Linux.
```

Comment:
Could you please list the submodule dependencies here, like we do for, e.g., `concordium-client`?

Reply:
I added the dependencies to the description.